.gitignore

```
@@ -0,0 +1,2 @@
+/node_modules
```

Нужно исправить: исправить игнорирование временных файлов.
В проекте есть [`/src/node_modules`](https://github.com/Mlenizyaka/Mesto_Webpack/tree/develop/src/node_modules), в которой те же самые временные файлы. Если убрать слэш в самом начале (`/`) то будут игнорироваться все директории `node_modules` не зависимо от них расположения в проекте.
```suggestion
node_modules
```

---

Дополнительно. Нужно удалить директорию [`/src/node_modules`](https://github.com/Mlenizyaka/Mesto_Webpack/tree/develop/src/node_modules) потому что она уже запушена в репозиторий.